lib/delayed_job_web/application/views/job.erb

```diff
@@ -60,5 +60,11 @@
         <%=h job.failed_at %>
       </dd>
     <% end %>
+    <% job.attribute_names.excluding('id', 'priority', 'attempts', 'queue', 'handler', 'last_error', 'run_at', 'locked_at', 'locked_by', 'failed_at').each do |column| %>
+      <dt><%= column %></dt>
+      <dd>
+        <%=h eval("job.#{column}") %>
+      </dd>
```

Comment:
I also saw that if the value is blank for the column, it breaks the layout. So, this should probably be:
```suggestion
      <% column_value = job.send(column) %>
      <% if column_value %>
        <dt><%= column %></dt>
        <dd>
          <%=h column_value %>
        </dd>
      <% end %>
```

Reply:
@markbeme Thoughts?